src/index.ts

```diff
@@ -1,4 +1,7 @@
 import {RlpDecode, RlpEncode, RlpItem, RlpList} from 'rlp-stream';
+import { sort } from 'shelljs';
```

Comment:
Where are these imports coming from? They seem to be unused.

Reply:
:') I'm not very sure, they are auto-generated. I guess it is because I used keys.sort() for radix sort.

Reply:
That doesn't really make sense, keys.sort is from the Array type..

Shelljs seems to be to execute unix commands? And that doesn't explain fs-extra or assert (which aren't used anywhere)...